generation.out

```diff
@@ -9,15 +9,15 @@ WARNING: Skipping resource type Microsoft.Addons/supportProviders/supportPlanTyp
 Processing batch task - {"tag":"schema-addons-2017-05-15"} .
 Resolving file file:///home/runner/work/bicep-types-az/bicep-types-az/workflow-temp/azure-rest-api-specs/specification/addons/resource-manager/Microsoft.Addons/preview/2017-05-15/Addons.json
 WARNING: Skipping resource type Microsoft.Addons/supportProviders/supportPlanTypes under path '/subscriptions/{subscriptionId}/providers/Microsoft.Addons/supportProviders/{providerName}/supportPlanTypes/{planTypeName}': No resource body defined
-[4.89 s] Generation Complete
+[4.9 s] Generation Complete
```

Comment:
Is this supposed to be checked in? (Looks like a log file.)

Reply:
Yeah, it's logged by the [Update Types pipeline](https://github.com/Azure/bicep-types-az/actions?query=workflow%3A%22Update+Types%22). I use it it to review diffs / warnings in case something unexpected has been broken in swagger. I'd like to make it a little less prone to unrelated changes (e.g. timing info), so there's less noise, but haven't got around to that yet.

Reply:
Makes sense.